[2023-03-01 13:10:29,757] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-09T05:05:00+00:00 [queued]>
[2023-03-01 13:10:29,987] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-09T05:05:00+00:00 [queued]>
[2023-03-01 13:10:29,989] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:29,990] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:10:29,991] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:10:30,185] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-12-09T05:05:00+00:00
[2023-03-01 13:10:30,234] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-12-09T05:05:00+00:00', '--job-id', '4946', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmplcry1t04', '--error-file', '/tmp/tmpcjsl65h1']
[2023-03-01 13:10:30,261] {standard_task_runner.py:77} INFO - Job 4946: Subtask get_name
[2023-03-01 13:10:30,207] {standard_task_runner.py:52} INFO - Started process 20641 to run task
[2023-03-01 13:10:30,829] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-12-09T05:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:10:31,473] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-12-09T05:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-09T05:05:00+00:00
[2023-03-01 13:10:31,605] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:10:31,737] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221209T050500, start_date=20230301T131029, end_date=20230301T131031
[2023-03-01 13:10:32,498] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:10:32,652] {local_task_job.py:146} INFO - Task exited with return code 0
